packages/playwright-test/src/util.ts

```diff
@@ -21,9 +21,35 @@ import type { TestError, Location } from './types';
 import { default as minimatch } from 'minimatch';
 import debug from 'debug';
 import { calculateSha1, isRegExp } from 'playwright-core/lib/utils/utils';
+import { isInternalFileName } from 'playwright-core/lib/utils/stackTrace';
+
+const PLAYWRIGHT_CORE_PATH = path.dirname(require.resolve('playwright-core'));
+const PLAYWRIGHT_TEST_PATH = path.join(__dirname, '..');
+
+function filterStackTrace(e: Error) {
```

Comment:
Short comment about this method would be nice:
- what it tries to achieve
- that it only works if error.stack was not touched before, which is the case for simple `throw` and `expect` statements.